content/en/docs/guides/metadata.md

```diff
@@ -116,4 +115,4 @@ the server.
 
 [Python Metadata]: https://github.com/grpc/grpc/tree/master/examples/python/metadata
 
-[Go Documentation]: https://github.com/grpc/grpc-go/blob/master/Documentation/grpc-metadata.md
```

Comment:
Why is this line changed? Did you delete or add a newline at the end of the file or something?  Github is hiding it from me.  (Please make sure there _is_ a newline.)

Reply:
Crap, looks like my editor is doing this by itself. What purpose does the newline at the end serve?

Reply:
Looks like there was no newline earlier at the end of the last line of the doc, and my editor was adding it. Got rid of it now.

Reply:
Oh.. OK.  Anything that cares (that I've ever seen) always _wants_ there to be a newline, but not changing it SGTM.